arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,130 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "sALifeTimeSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association lifetime in seconds for a site to site VPN tunnel."
+        },
+        "sADataSizeKilobytes": {
```

Comment:
@olydis Ok thanks, I checked the actual responses. False alarm.
@amarzavery Is there any reason why NRP differentiates the actual request properties from the server model?